server/views/pages/reviewPlan/checkYourAnswers.njk

```diff
@@ -12,7 +12,75 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">Check your answers</h1>
+      <h1 class="govuk-heading-l" data-qa="page-heading">Check
+        {{prisonerSummary.firstName + " " + prisonerSummary.lastName}}'s review information</h1>
+
+      <div class="app-u-heading-with-change-link">
+        <h2 class="govuk-heading-m">
+          {% if reviewPlanDto.completedBy === 'MYSELF' %}
+            Who completed the review?
+          {% else %}
+            Who completed
+            {{ prisonerSummary.firstName + " " + prisonerSummary.lastName }}'s review?
+          {% endif %}
+        </h2>
+        <span class="govuk-body-m">
+          <a class="govuk-link" href="/plan/{{ prisonerSummary.prisonNumber }}/review" data-qa="review-completed-by-change-link" style="white-space: nowrap;">
+            Change<span class="govuk-visually-hidden">
+              who completed the review</span>
+          </a>
+        </span>
+      </div>
+
+      <dl class="govuk-summary-list govuk-!-margin-bottom-8">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key govuk-!-width-one-quarter">Review completed by</dt>
+          <dd class="govuk-summary-list__value" data-qa="review-completed-by">
+            {% if reviewPlanDto.completedBy === 'MYSELF' %}
+              {{ userName }}
+            {% else %}
+              {{ reviewPlanDto.completedByOtherFullName }}
+            {% endif %}
+          </dd>
+        </div>
+        {% if reviewPlanDto.completedBy === 'SOMEBODY_ELSE' %}
+          <div class="govuk-summary-list__row">
+            <dt class="govuk-summary-list__key govuk-!-width-one-quarter">Job role</dt>
+            <dd class="govuk-summary-list__value" data-qa="job-role">
+              {{ reviewPlanDto.completedByOtherJobRole }}
+            </dd>
+          </div>
+        {% endif %}
+      </dl>
+      <div class="app-u-heading-with-change-link">
+        <h2 class="govuk-heading-m">Review note</h2>
+        <span class="govuk-body-m">
+          <a class="govuk-link" href="/plan/{{ prisonerSummary.prisonNumber }}/review/notes" data-qa="review-note-change-link" style="white-space: nowrap;">
+            Change<span class="govuk-visually-hidden">
+              who completed the review</span>
```

Comment:
Minor - the visually hidden text is wrong here - this is the change link for the note 

Also, what is the inline style for white-space: nowrap for? (and in the change link above)

I think we should probably look to avoid inline styles 🤔 , but I'm wondering why we need it at all? Is it fixing a layout problem that we can perhaps fix some other way? Do our other check your answers pages have similar problems with their change links? 🤔

Reply:
Have a look at the Education & Training tab , specifically the Educational Qualifications heading and change link:
![Screenshot 2024-11-01 at 09 03 08](https://github.com/user-attachments/assets/2d8e7908-0970-4a37-8754-ad5d4eb89b01)

I think we've done essentially the same thing there, but without inline css:
```
            <div class="app-summary-card__change-link">
              <h3 class="govuk-heading-s app-summary-card__change-link__heading">Educational qualifications</h3>
              {%- set educationalQualificationsChangeLinkUrl -%}
                {# The target of the change link is different depending on whether the education record has qualifications already recorded on not #}
                {%- if education.educationDto.qualifications | length -%}
                  {# If the education record has qualifications the change link should be to the qualifications list screen, which allows the user to add/remove qualifications #}
                  /prisoners/{{ prisonerSummary.prisonNumber }}/education/qualifications
                {%- else -%}
                  {# If the education record does not have any qualifications the change link should be to a route that sets up the DTO in the prisoner context then redirects to the qualification level screen, which allows the user enter the first qualification #}
                  /prisoners/{{ prisonerSummary.prisonNumber }}/education/add-qualifications
                {%- endif -%}
              {%- endset -%}
              <a class="govuk-link app-summary-card__change-link__link govuk-!-display-none-print" href="{{ educationalQualificationsChangeLinkUrl }}" data-qa="educational-qualifications-change-link">
                Change<span class="govuk-visually-hidden"> qualifications by adding or removing qualifications</span>
              </a>
            </div>
```
The above has some complexity you wont need re: working out the URL to link to; and the above example is a h3 heading whereas you have yours as a h2 (which I think is correct for this page 👍 )
But you get the idea, and hopefully we can apply it to this page too 👍 🤞 
Basically:
* we have a parent `div` with class `app-summary-card__change-link`
* the heading element has a class `app-summary-card__change-link__heading`
* the change link anchor tag element has a class `app-summary-card__change-link__link`

Hopefully that's all you need 🤞

Reply:
Refactored to use the formatting above, negating the need for inline style.